x-pack/plugins/index_management/public/plugin.ts

```diff
@@ -98,29 +99,34 @@ export class IndexMgmtUIPlugin
     coreSetup: CoreSetup<StartDependencies>,
     plugins: SetupDependencies
   ): IndexManagementPluginSetup {
-    if (this.config.isIndexManagementUiEnabled) {
```

Comment:
This config was provided to users in order to allow them to disable the index management UI if they wanted through xpack.index_management.enabled: false. I think we should retain that functionality as it might be part of a bigger change.